packages/webdriverio/src/utils/interception/utils.ts

```diff
@@ -25,7 +25,7 @@ export function parseOverwrite<
              */
             {
                 type: 'string',
-                value: bodyOverwrite
+                value: base64Decode(base64Encode(bodyOverwrite))
```

Comment:
Why to we encode and then decode the body?

Reply:
I was trying many ways like using `JSON.stringify`, `btoa` and so many more but none of them worked.


This works by encoding the strings into an UTF8 array and calculating the width then decoding it back into a string.

If you know better solutions, please let me know